colcon_cmake/task/cmake/__init__.py

```diff
@@ -65,6 +65,11 @@ async def has_target(path, target):
         "'has_target' not implemented for CMake generator '{generator}'" \
         .format_map(locals())
 
+async def get_makefile_has_targets(path, target):
+    output = subprocess.run([
```

Comment:
This should probably use an "awaitable" function, like `await check_output(...)` that `get_makefile_targets()` uses.

Reply:
replaced by asyncio.create_subprocess_exec